xgboost_trainer.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #137.        best_params = perform_grid_search(train_pairs, cv_pairs)
</pre><p>would add a comment here - something like "Perform grid search to determine optimal hyper parameters for training"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>